app/assets/javascripts/global-bar-init.js

```diff
@@ -26,12 +26,14 @@ var globalBarInit = {
 
   blacklistedUrl: function() {
     var paths = [
-      "/register-to-vote",
-      "/done",
-      "/transition",
-      "/transition-check"
```

Comment:
Please leave `/transition-check` in the blacklist. The others (Transition landing page, and RtV start page) can come out (so the banner shows on them).